src/main/util.ts

```diff
@@ -59,19 +59,20 @@ const loadAllVideos = (storageDir: any, videoState: any) => {
  const loadVideoDetails = (video: any, videoState: any) => {
     const dateObject = new Date(fs.statSync(video.name).mtime)
     const metadata = getMetadataForVideo(video)
-
-    videoState[metadata.category].push({
-        index: videoIndex[metadata.category]++,
-        fullPath: video.name,
-        zone: getVideoZone(metadata.zoneID, metadata.category),
-        zoneID: metadata.zoneID,
-        encounter: getVideoEncounter(metadata),
-        encounterID: metadata.encounterID,
-        duration: metadata.duration,
-        result: metadata.result, 
-        date: getVideoDate(dateObject),
-        time: getVideoTime(dateObject)
-    });
+    if (metadata != undefined) {
```

Comment:
I think you want `!==` here (see https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a).

Reply:
interesting. didnt even know that existed. 
```suggestion
    if (metadata !== undefined) {
```